modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/enumerations.scala

```diff
@@ -158,4 +158,30 @@ object enumerations {
       "CUSTOM_MASK" -> "Custom Mask"
     )
   }
+
+  object disperser {
+    val GmosSDisperser: Map[String, String] = Map(
+      "MIRROR"      -> "Mirror",
+      "B1200_G5321" -> "B1200",
```

Comment:
Do we display the right-hand side right? Science wanted the names without the gemini code

Reply:
Yes, we only display the right hand side names. We look up those names with the left hand side and translate to the right hand side.